chapter_6_practic/task5.cpp

```diff
@@ -0,0 +1,39 @@
+// Start with the date structure in Exercise 5 in Chapter 4 and transform it into a date
+// class.Its member data should consist of three ints : month, day, and year.It should also
+// have two member functions : getdate(), which allows the user to enter a date in
+// 12 / 31 / 02 format, and showdate(), which displays the date.
+
+#include <iostream>
+#include <iomanip>
+
+class date
+{
+private:
+	int day;
+	int month;
+	int year;
+	char ch;
```

Comment:
Why do you need to keep 'ch' as a member?

Reply:
Every instance of this class will keep additional 'ch' char. Whereas, this member is needed only locally, in the get function.

Reply:
Fixed
https://github.com/Dmitriy-Romanchuk/master/pull/6/commits/6ce145bb54e2fbbd03b277137bf879c738f68190